migrations/V5__Add_Business_Case_Table.sql

```diff
@@ -0,0 +1,52 @@
+create table business_case (
+   id uuid PRIMARY KEY not null,
+   eua_user_id text not null,
+   project_name text,
+   requester text,
+   requester_phone_number text,
+   business_owner text,
+   business_need text,
+   cms_benefit text,
+   priority_alignment text,
+   success_indicators text,
+
+   as_is_title text,
+   as_is_summary text,
+   as_is_pros text,
+   as_is_cons text,
+   as_is_cost_savings text,
+
+   preferred_title text,
```

Comment:
Can there be more than two alternatives? If so, maybe better to break these chunks out into their own table with `title, summary, acquisition approach, pros, cons, cost savings`

Reply:
I think it's just 2 alternative approaches at least in the UI. @christopherhuii do you know something different?

Reply:
Only 2 according to design and prd: https://trussworks.slack.com/archives/CLCFT3EPM/p1588705171176700?thread_ts=1588705122.176500&cid=CLCFT3EPM

Reply:
Okay, sounds good.